src/test/unit/web_interface/rest/test_rest_binary_search.py

```diff
@@ -3,7 +3,7 @@
 
 def test_no_data(test_client):
     response = test_client.post('/rest/binary_search')
-    assert response.status_code == 400
+    assert response.status_code == 415
```

Comment:
Must these status codes be hardcoded?

Reply:
Not really. I replaced them with values from `http.HTTPStatus`